src/cf-pagination/src/molecules/pagination.less

```diff
@@ -0,0 +1,107 @@
+//
+// Theme variables
+//
+
+// Color variables
+// $color- variables are from 18F's US Web Design Standards
+// https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_variables.scss
+
+@pagination-text-color: #757575; // $color-gray-medium
+@pagination-bg-color: #f1f1f1; // $color-gray-lightest
+
+// Sizing variables
+
+@pagination-font-size__px: @base-font-size-px;
+@pagination-font-size__em: unit( @pagination-font-size__px / @base-font-size-px, em );
+@pagination-height__px: 42px;
+@pagination-width__px: 130px;
+
+//
+// Pagination
+//
+
+.m-pagination {
+    position: relative;
+
+    &_form {
+        padding: unit( 5px / @base-font-size-px, em );
+
+        border-radius: unit( 4px / @base-font-size-px, em );
+        background: @pagination-bg-color;
+        color: @pagination-text-color;
+        text-align: center;
+    }
+
+    // Currently requires explicit input[type="number"] due to cascade issue
+    // TODO: Fix cascade issue, possibly due to import order
+    input[type="number"],
+    &_current-page {
+        .webfont-medium();
+
+        padding-top: unit( 5.5px / @base-font-size-px, em );
+        padding-bottom: unit( 5.5px / @base-font-size-px, em );
+
+        // We need enough room for three digits and the
+        // number spinners for type="number" inputs.
+        width: unit( 42px / @pagination-font-size__px, em );
+        margin: 0 @pagination-font-size__em / 2;
+
+        text-align: right;
+        vertical-align: middle;
+    }
+
+    &_label {
+        display: inline-block;
+
+        margin-bottom: 0;
+        margin-right: @pagination-font-size__em / 2;
+
+        vertical-align: middle;
+    }
+
+    &_submit-btn {
```

Comment:
Shouldn't it describe the action, and since it's a form, that'd be `_btn-submit`